src/JustSaying/AwsTools/MessageHandling/MessageBatchResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net;
+using Amazon.Runtime;
+using JustSaying.Messaging;
+
+namespace JustSaying.AwsTools.MessageHandling;
+
+public class MessageBatchResponse
+{
+    public IEnumerable<string> SuccessfulMessageIds { get; set; }
+    public IEnumerable<string> FailedMessageIds { get; set; }
```

Comment:
Prefer `ICollection<T>` or `IList<T>` as it guards against things like LINQ queries being assigned that then get enumerated multiple times.